### PR TITLE
Add /content/kotlin/concepts/operators/operators.md first version file

### PR DESCRIPTION
**Pull request by JuanGabrielTerrazasSarmiento**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/operators/operators.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+val f = a / b // f is now 2
+val g = a % b // g is now 0
+```
+
+## Comparison Operators
+
+Comparison operators are used to compare two values and determine wheter they are equal, not equal, greater than, less than, greater than or equal to, or less than or equal to. Kotlin supports the following comparison operators:
+
+* '==' (equals to)
+* '!=' (not equals to)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:46]:
```suggestion
* '!=' (not equal)
```

File: content/kotlin/concepts/operators/operators.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+* '>' (greater than)
+* '<' (less than)
+* '>=' ()greater than or equals to)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:49]:
```suggestion
* '>=' (greater than or equal to)
```

File: content/kotlin/concepts/operators/operators.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* '<=' (less than or equals to)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:50]:
```suggestion
* '<=' (less than or equal to)
```

File: content/kotlin/concepts/operators/operators.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+println(a != b) // true
+println(a > b) // true
+println(a < b) // false
+println(a >= c) // true
+println(d <= b) // false
+```
+
+## Logical Operators
+
+Logical operators are used to combine two or mode conditions and determine the overall result. Kotlin supports the following logical operators:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:70]:
```suggestion
Logical operators are used to combine two or more conditions and determine the overall result. Kotlin supports the following logical operators:
```

File: content/kotlin/concepts/operators/operators.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+println(b.inv()) // 0b0011
+```
+
+## Assignment Operators
+
+Assignment operators are used to assign a value to a variable. Kotlin supports the following assignment operators:
+
+* '=' (simple assignment) used to specify default values for parameters.
+* '+=' (addition assignment)
+* '-=' (substraction assignment)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:115]:
```suggestion
* '-=' (subtraction assignment)
```

File: content/kotlin/concepts/operators/operators.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+* '*=' (multiplication assignment)
+* '/*' (division assignment)
+* '%=' (module assignment)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:118]:
```suggestion
* '%=' (modulus assignment)
```

File: content/kotlin/concepts/operators/operators.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+## Increment and decrement operators
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:120]:
```suggestion
## Increment and Decrement Operators
```

File: content/kotlin/concepts/operators/operators.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+Here's an example of how to use bitwise operators in Kotlin:
+
+```
+val a = 0b1010
+val b = 0b1100
+
+println(a and b) // 0b1000
+println(a or b) // 0b1110
+println(a xor b) // 0b0110
+println(b.inv()) // 0b0011
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/operators/operators.md:106]: I believe this prints out integers

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@JuanGabrielTerrazasSarmiento I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by caupolicandiaz** [comment]
@JuanGabrielTerrazasSarmiento There hasn't been any action on this PR, do you need any assistance with the changes?

**Review by caupolicandiaz — APPROVED** [review]
@JuanGabrielTerrazasSarmiento I've gone ahead and made the changes I suggested in order to keep this entry moving.

File: content/kotlin/concepts/operators/operators.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+Tags:
+  - 'Operators'
+  - 'Conditionals'
+  - 'Kotlin'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+## Understanding Kotlin Operators
```
Comment (SSwiniarski) [inline content/kotlin/concepts/operators/operators.md:16]: We generally don't start with a header, so remove this one.

File: content/kotlin/concepts/operators/operators.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+val c = a + b // c is now 15
+val d = a - b // d is now 5
+val e = a * b // e is now 50
+val f = a / b // f is now 2
+val g = a % b // g is now 0
+```
+
+## Comparison Operators
+
+Comparison operators are used to compare two values and determine wheter they are equal, not equal, greater than, less than, greater than or equal to, or less than or equal to. Kotlin supports the following comparison operators:
```
Comment (SSwiniarski) [inline content/kotlin/concepts/operators/operators.md:46]:
```suggestion
Comparison operators are used to compare two values and determine whether they are equal, not equal, greater than, less than, greater than or equal to, or less than or equal to. Kotlin supports the following comparison operators:
```

File: content/kotlin/concepts/operators/operators.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+
+- '==' (equals to)
+- '!=' (not equals to)
+- '>' (greater than)
+- '<' (less than)
+- '>=' ()greater than or equals to)
```
Comment (SSwiniarski) [inline content/kotlin/concepts/operators/operators.md:52]:
```suggestion
- '>=' (greater than or equals to)
```

File: content/kotlin/concepts/operators/operators.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+val a = 0b1010
+val b = 0b1100
+
+println(a and b) // 8
+println(a or b) // 14
+println(a xor b) // 6
+println(b.inv()) // -13
+```
+
+## Assignment Operators
```
Comment (SSwiniarski) [inline content/kotlin/concepts/operators/operators.md:112]: You're missing examples for the last two sections.

File: content/kotlin/concepts/operators/operators.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+```
+
+## Bitwise Operators
+
+Bitwise operators are used to perform bitwise operations on binary values. Kotlin supports the following bitwise operators:
+
+- 'and' (bitwise and)
+- 'or' (bitwise or)
+- 'xor' (bitwise xor)
+- 'inv' (bitwise complement)
```
Comment (SSwiniarski) [inline content/kotlin/concepts/operators/operators.md:98]: `inv()` is a function, not an operator.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@JuanGabrielTerrazasSarmiento I've done a second review and left a few comments. 

One additional thing I'd like to see is a sentence about what each operator does as part of the bulleted lists. so they read something like this:

```
- `+` Addition: Adds the left argument to the right argument.
```

Also I'd like the lists edited as above, with the operator in single backticks, removing the parenthesis, and capitalizing the operator name.

**Comment by caupolicandiaz** [comment]
@JuanGabrielTerrazasSarmiento There hasn't been any updates on this entry in the past week, are you still working on this? Please advise if you need any assistance?

**Comment by caupolicandiaz** [comment]
@SSwiniarski @JuanGabrielTerrazasSarmiento I've gone ahead and incorporated the edits from the second review.

**Review by SSwiniarski — APPROVED** [review]
@JuanGabrielTerrazasSarmiento Looks good to me.

**Comment by SSwiniarski** [comment]
@JuanGabrielTerrazasSarmiento congrats on your new entry. Live link is here: https://www.codecademy.com/resources/docs/kotlin/operators

@yangc95, @KTom101, @caupolicandiaz, This is another one where the user isn't being credited with the entry.
  
image

**Comment by yangc95** [comment]
@JuanGabrielTerrazasSarmiento I would still recommend following the recommended steps to ensure that:

1. Your e-mails are consistent with your Codecademy profile and GitHub (then log back in to Codecademy with the GitHub option)
2, Your Codecademy profile is visible to everyone in your privacy settings and has the correct reference to your GitHub username

**Pull request merged**